playground/css/__tests__/css.spec.ts

```diff
@@ -292,6 +292,10 @@ test('@import dependency that @import another dependency', async () => {
   expect(await getColor('.css-proxy-dep')).toBe('purple')
 })
 
+test('@import scss dependency that @import another dependency', async () => {
```

Comment:
```suggestion
test('@import scss dependency that has @import with a css extension pointing to another dependency', async () => {
```
Let's clarify that the @import needs `.css`.